docs/_docs/contributing/getting-started.md

```diff
@@ -43,7 +50,6 @@ Start by cloning the repository:
 ```bash
 $ git clone https://github.com/lampepfl/dotty.git
 $ cd dotty
-$ sbt managedSources # Needed for IDE import to succeed
```

Comment:
I've never used this and testing with Metals, this doesn't seem to be necessary.

Reply:
Excellent! Reading the current version, I was just going to say we should remove this!